ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/BlockchainQueries.java

```diff
@@ -480,6 +480,10 @@ private TransactionWithMetadata transactionByHeaderAndIndex(
         txs.get(txIndex), header.getNumber(), blockHeaderHash, txIndex);
   }
 
+  public Optional<TransactionLocation> transactionLocationByHash(final Hash transactionHash) {
```

Comment:
I don't really like that way of naming methods based on their arguments. But I can see that the other method names are the same ...